tests/unit/test_fs_funcs.py

```diff
@@ -0,0 +1,19 @@
+"""Regression tests for bugtool helper functions like get_recent_logs()"""
+
+
+def test_get_recent_logs(bugtool):
+    """Assert get_recent_logs() returning the most recent file(s)"""
+
+    files = ["/proc/1/stat", "/proc/self/stat"]  # self/stat is more recent.
+    # return the files, ordered by their modification time, most recent first:
+    assert bugtool.get_recent_logs(files, 9) == files
+    assert bugtool.get_recent_logs([files[1], files[0]], 9) == files
+    # When verbosity is 1, only return the most recent file:
+    assert bugtool.get_recent_logs(files, 1) == [files[1]]
+
+
+def test_read_inventory(bugtool, dom0_template):
+    """Assert readKevValueFile() reading the a xensource inventory file"""
```

Comment:
Key misspelled

Reply:
Fixed, rebased to the updated master branch, added more fixes and enabled the merged tests.

Ready for review again, but I can also split the commits if needed.